### ISSUE TITLE
Can't verify the app since I don't own the flycast.org domain

### ISSUE THREAD

**Issue by flyinghead** [flathub/org.flycast.Flycast#201]
I was asked to go through the Flathub verification process to certify the app's origin. See https://github.com/flyinghead/flycast/issues/1473.
I'd be happy to do so but Flathub assumes I own the app id domain (`flycast.org`) which I don't. I do own the `flyca.st` domain but ideally a github id would be more future-proof (`io.github.flyinghead.flycast`).

Either way it looks like we need to change the app id to get it verified.

**Comment by j8r** [comment]
Yes I did the mistake of not using the proper ID when creating the application.

I think the best way may be to rename the application.

I'm not sure about the procedure, I can see this doc: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#replaces.

We may have to contact admins to rename the repository.

**Comment by Roman19S** [comment]
Is there an update on this? Distros like Linux Mint and Elementary OS have started hiding or otherwise discouraging the use of unverified flatpaks. More are likely to follow as verified flatpak's near 50% of Flathub. I would hate for people to miss out on Flycast because of this
